src/lib/components/wallet/CreateTransaction.svelte

```diff
@@ -26,19 +20,40 @@
             ConversationStore.addPendingMessages(chat.id, res.message, txt)
         })
     }
+
+    let display_amount = ""
+    function onChangeAmount() {
+        wallet.get_amount_display(transfer.asset, transfer.amount).then(display => {
```

Comment:
I usually prefer to await, but in this case, `function onChangeAmount()` and `function onChangeAssetKind()` can't be async. This is because they are called via `on:change={onChangeAssetKind}` in `<Select>` and `on:change={onChangeAmount}` in `Input`
So, instead of awaiting, I called `.then()` which allows you to provide a callback function that is called after the promise completes